client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -195,76 +202,113 @@ const PluginsBrowser = React.createClass( {
 		);
 	},
 
-	getSearchBox( pinned ) {
-		if ( pinned ) {
-			return (
-				<Search
-					pinned
-					fitsContainer
-					onSearch={ this.doSearch }
-					initialValue={ this.props.search }
-					placeholder={ this.translate( 'Search Plugins' ) }
-					delaySearch={ true }
-					analyticsGroup="PluginsBrowser" />
-			);
-		}
+	getSearchBox() {
+		const { WrappedSearch } = this;
 
 		return (
-			<SearchCard
-				autoFocus={ ! hasTouch() }
+			<WrappedSearch
+				pinned
+				fitsContainer
 				onSearch={ this.doSearch }
 				initialValue={ this.props.search }
-				placeholder={ this.translate( 'Search Plugins' ) }
+				placeholder={ this.props.translate( 'Search Plugins' ) }
 				delaySearch={ true }
-				analyticsGroup="PluginsBrowser" />
+				analyticsGroup="PluginsBrowser"
+			/>
 		);
 	},
 
 	getNavigationBar() {
 		const site = this.props.site ? '/' + this.props.site : '';
-		return <SectionNav selectedText={ this.translate( 'Category', { context: 'Category of plugins to be filtered by' } ) }>
+		return <SectionNav selectedText={ this.props.translate( 'Category', { context: 'Category of plugins to be filtered by' } ) }>
 			<NavTabs label="Category">
 				<NavItem
 					path={ '/plugins/browse' + site }
 					selected={ false }
 				>
-					{ this.translate( 'All', { context: 'Filter all plugins' } ) }
+					{ this.props.translate( 'All', { context: 'Filter all plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/featured' + site }
 					selected={ this.props.path === ( '/plugins/browse/featured' + site ) }
 				>
-					{ this.translate( 'Featured', { context: 'Filter featured plugins' } ) }
+					{ this.props.translate( 'Featured', { context: 'Filter featured plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/popular' + site }
 					selected={ this.props.path === ( '/plugins/browse/popular' + site ) }
 				>
-					{ this.translate( 'Popular', { context: 'Filter popular plugins' } ) }
+					{ this.props.translate( 'Popular', { context: 'Filter popular plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/new' + site }
 					selected={ this.props.path === ( '/plugins/browse/new' + site ) }
 				>
-					{ this.translate( 'New', { context: 'Filter new plugins' } ) }
+					{ this.props.translate( 'New', { context: 'Filter new plugins' } ) }
 				</NavItem>
 			</NavTabs>
-			{ this.getSearchBox( true ) }
+			{ this.getSearchBox() }
 		</SectionNav>;
 	},
 
-	getPageHeaderView() {
-		if ( this.props.category ) {
-			return this.getNavigationBar();
-		}
+	handleSuggestedSearch( term ) {
+		this.reinitializeSearch();
+		return () => this.doSearch( term );
+	},
 
+	getSearchBar() {
+		const suggestedSearches = [
+			this.props.translate( 'Engagement', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Security', { context: 'Plugins suggested search term' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Security', { context: 'settings screen'} )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/add-plugins-page-first)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?